eventlogger/pom.xml

```diff
@@ -73,6 +73,7 @@
         <dependency>
             <groupId>org.postgresql</groupId>
             <artifactId>postgresql</artifactId>
+            <version>42.4.0</version>
```

Comment:
It looks like the version is not specified any any of the postgresql dependencies. Create a variable in the top level pom for the version and propagate it down to all the poms.